src/components/Auth/assets/animations.scss

```diff
@@ -1,3 +1,4 @@
+/* stylelint-disable function-parentheses-space-inside */
```

Comment:
This needs to be re-enabled at the end of the file since we should only be temporarily disabling this rule for this file. See https://stylelint.io/user-guide/ignore-code for more info.